src/packages/@ncigdc/modern_components/Lolliplot/separateOverlapping.js

```diff
@@ -0,0 +1,18 @@
+export const overlaps = (a, b) => {
+  const first = a.start < b.start ? a : b;
+  const last = first.start === a.start ? b : a;
+  return first.end > last.start;
+};
+
+const separateOverlapping = data => {
+  return data.slice(1).reduce((acc, d) => {
```

Comment:
Would probably be good to remove the slice and set the initial value to [], I don't think it changes the output at all?